R/toml_serialization.R

```diff
@@ -122,7 +123,8 @@ format_toml.data.frame <- function(x,
                                    .tbl_name,
                                    .top_level = FALSE) {
   rows <- nrow(x)
-  header <- glue("[[{.tbl_name}]]")
+  # header <- glue("[[{.tbl_name}]]")
```

Comment:
If you aim at throwing away `glue`, why comment and not remove?

Reply:
ah good catch, i was trying to find an error that was coming from glue and I wanted to see if it was from here. This can be either removed or reverted to glue. Only focus in this PR is for purrr at the moment.